### PR TITLE
[CELEBORN-1263] Fix Master HA mode without internal port error

### PR DESCRIPTION
**Pull request by wxplovecc**
### What changes were proposed in this pull request?
Master can not start up with ha mode

### Why are the changes needed?
image

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
local tested

### CODE REVIEW

File: master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala [hunk 1]
```diff
@@ -44,7 +44,9 @@ class MasterArguments(args: Array[String], conf: CelebornConf) {
     val localNode = clusterInfo.localNode
     _host = _host.orElse(Some(conf.haMasterNodeHost(localNode.nodeId)))
     _port = _port.orElse(Some(conf.haMasterNodePort(localNode.nodeId)))
-    _internalPort = _internalPort.orElse(Some(conf.haMasterNodeInternalPort(localNode.nodeId)))
+    _internalPort = _internalPort.orElse(if (conf.internalPortEnabled)
+      Some(conf.haMasterNodeInternalPort(localNode.nodeId))
+    else None)
```
Comment (pan3793) [inline master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala:49]: is the following style allowed by our scalafmt rules?
```suggestion
    _internalPort = _internalPort.orElse { 
      if (conf.internalPortEnabled) Some(conf.haMasterNodeInternalPort(localNode.nodeId)) else None
    }
```

File: master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala [hunk 2]
```diff
@@ -44,7 +44,9 @@ class MasterArguments(args: Array[String], conf: CelebornConf) {
     val localNode = clusterInfo.localNode
     _host = _host.orElse(Some(conf.haMasterNodeHost(localNode.nodeId)))
     _port = _port.orElse(Some(conf.haMasterNodePort(localNode.nodeId)))
-    _internalPort = _internalPort.orElse(Some(conf.haMasterNodeInternalPort(localNode.nodeId)))
+    _internalPort = _internalPort.orElse {
```
Comment (pan3793) [inline master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala:47]:
```suggestion
    _internalPort = _internalPort.orElse {
```

Comment (pan3793) [reply master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala:47]: my bad, there is a extra space at the end

Comment (wxplovecc) [reply master/src/main/scala/org/apache/celeborn/service/deploy/master/MasterArguments.scala]: +1

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to main for 0.5

**Pull request closed without merging**